Source/ACE/Network/GameAction/Actions/GameActionDie.cs

```diff
@@ -6,7 +6,7 @@ public static class GameActionDie
         [GameAction(GameActionType.EvtCharacterSuicide)]
         public static void Handle(ClientMessage message, Session session)
         {
-            session.Player.Kill();
+            session.Player.Kill(session.Player.Guid);
```

Comment:
Should enqueue a GameAction here so the landblock can do the corpse-dropping logic.